structiagram.md

```diff
@@ -0,0 +1,1234 @@
+# Structiagram
```

Comment:
What is this? It doesn't seem to be valid mermaid code.

Reply:
That was lingering in my repo since the storage trait work and it seems I accidentally committed it. I removed it.